stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/channel/logic/ChannelLogic.kt

```diff
@@ -153,6 +153,7 @@ internal class ChannelLogic(
         setWatchers(c.watchers)
         upsertMessages(c.messages)
         mutableState.lastMessageAt.value = c.lastMessageAt
+        mutableState.channelConfig.value = c.config
```

Comment:
Added it to be a memory state flow. Why do we need go to DB if we sync memory data in channel controller always?